### PR TITLE
Bugfix/build fix

### PR DESCRIPTION
**Pull request by div-Leo**
**What:**
Fix docz build

**Why:**
It was not running with the latest versions of node.
​
**How:**

- Added `NODE_OPTIONS=--openssl-legacy-provider` flag 
- recompiled `package-lock.json`
- cleaned the .docz cache​

### CODE REVIEW

**Review by jonotrujillo — APPROVED** [review]
Thanks for fixing it!

**Review by martimalek — APPROVED** [review]

File: src/components/Table/components/Table.tsx [hunk 1]
```diff
@@ -41,6 +46,7 @@ const Table: FC<TableProps> = ({
     };
 
     return (
+        // @ts-expect-error table props (height, width) conflict with HeightProps and WidthProps from styled-system
```
Comment (nlopin) [inline src/components/Table/components/Table.tsx:49]: I think it should be `ts-ignore`

Comment (lloydaf) [reply src/components/Table/components/Table.tsx]: should we update TS to 3.9 instead?

Comment (lloydaf) [reply src/components/Table/components/Table.tsx]: https://devblogs.microsoft.com/typescript/announcing-typescript-3-9/#ts-expect-error-comments

**Review by nlopin — APPROVED** [review]

File: src/components/Table/components/Table.tsx (hunk 1 above)
Comment (nlopin) [reply src/components/Table/components/Table.tsx]: I'll do it in a follow-up

**Pull request merged**